### PR TITLE
print output on test failure for t0081

### PR DESCRIPTION
**Pull request by whyrusleeping**
seeing another random failure that I may be responsible for here: https://github.com/ipfs/go-ipfs/issues/1508

This should help figure out whats going on.

Also removed a debug statement i accidentally left in.

License: MIT
Signed-off-by: Jeromy <jeromyj@gmail.com>

### CODE REVIEW

**Comment by whyrusleeping** [comment]
mmm, those test failures really make me want https://github.com/ipfs/go-ipfs/pull/1501 in even more

**Comment by jbenet** [comment]
I think this is unrelated to #1501

**Comment by whyrusleeping** [comment]
@jbenet the travis test failures are exactly what i fixed in #1501, what do you mean they are unrelated?

File: core/commands/pin.go [hunk 1]
```diff
@@ -60,11 +60,6 @@ on disk.
 			recursive = false
 		}
 
-		go func() {
-			<-req.Context().Done()
-			log.Error("CONTEXT IS OVER!")
```
Comment (jbenet) [inline core/commands/pin.go]: wat.

**Pull request merged**